doc/whatsnew/fragments/8280.false_negative

```diff
@@ -0,0 +1,3 @@
+Fix false negative for inconsistent-returns with while-loops.
+
+Closes 8280
```

Comment:
I forgot the `#` for the issue number. Is there a way to add that without rerunning all the tests?

Reply:
No :D